icons/last-dot-fm.svg

```diff
@@ -1 +1 @@
-<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Last.fm icon</title><path d="M10.599 17.211l-.881-2.393s-1.433 1.596-3.579 1.596c-1.9 0-3.249-1.652-3.249-4.296 0-3.385 1.708-4.596 3.388-4.596 2.418 0 3.184 1.568 3.845 3.578l.871 2.751c.871 2.672 2.523 4.818 7.285 4.818 3.41 0 5.722-1.045 5.722-3.801 0-2.227-1.276-3.383-3.635-3.935l-1.757-.384c-1.217-.274-1.577-.771-1.577-1.597 0-.936.736-1.487 1.952-1.487 1.323 0 2.028.495 2.147 1.679l2.749-.33c-.225-2.479-1.937-3.494-4.745-3.494-2.479 0-4.897.936-4.897 3.934 0 1.873.902 3.058 3.185 3.605l1.862.443c1.397.33 1.863.916 1.863 1.713 0 1.021-.992 1.441-2.869 1.441-2.779 0-3.936-1.457-4.597-3.469l-.901-2.75c-1.156-3.574-3.004-4.896-6.669-4.896C2.147 5.327 0 7.879 0 12.235c0 4.179 2.147 6.445 6.003 6.445 3.108 0 4.596-1.457 4.596-1.457v-.012z"/></svg>
\ No newline at end of file
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Last.fm icon</title><path d="M10.5986 17.2108l-.881-2.393s-1.433 1.596-3.5789 1.596c-1.8999 0-3.2488-1.652-3.2488-4.2958 0-3.3849 1.708-4.5958 3.3878-4.5958 2.418 0 3.184 1.568 3.8449 3.5778l.871 2.751c.871 2.6718 2.5228 4.8177 7.2846 4.8177 3.4099 0 5.7218-1.045 5.7218-3.8008 0-2.227-1.276-3.3829-3.6348-3.9349l-1.757-.384c-1.217-.274-1.577-.771-1.577-1.5969 0-.936.736-1.487 1.952-1.487 1.323 0 2.028.495 2.147 1.679l2.7488-.33c-.225-2.4789-1.937-3.4938-4.7448-3.4938-2.4789 0-4.8968.936-4.8968 3.9338 0 1.873.902 3.0579 3.1849 3.6049l1.8619.443c1.397.33 1.863.916 1.863 1.7129 0 1.021-.992 1.441-2.869 1.441-2.7788 0-3.9358-1.457-4.5968-3.469l-.901-2.7498c-1.1559-3.5739-3.0038-4.8958-6.6687-4.8958C2.147 5.3273 0 7.8792 0 12.235c0 4.1788 2.147 6.4447 6.0027 6.4447 3.108 0 4.5959-1.457 4.5959-1.457z"/></svg>
```

Comment:
Slight difference between PR (red) and [Wikipedia SVG](https://upload.wikimedia.org/wikipedia/commons/d/d4/Lastfm_logo.svg) - any chance you could take a look? Looks like the source link no longer works, so no 'official' SVG...
![image](https://user-images.githubusercontent.com/1830867/103038589-ba544e00-4566-11eb-8de1-3a3da2f1eae1.png)

Reply:
@adamrusted Done 🎧  I've changed the `source` as well.